components/AuthForm.tsx

```diff
@@ -41,7 +41,8 @@ const AuthForm: React.FC<AuthFormProps> = ({ type, onClose }) => {
         life: 3000,
       });
       //onClose();
-      window.location.reload();
+
+      window.location.href = "/";
```

Comment:
This will cause to go to Networks Status, instead of staying at the same page (e.g. Account Portal)